main/streams/streams.c

```diff
@@ -1555,6 +1555,81 @@ PHPAPI zend_result _php_stream_copy_to_stream_ex(php_stream *src, php_stream *de
 		return SUCCESS;
 	}
 
+#ifdef __linux__
```

Comment:
would it be possible to detect the support at configure time instead (ie freebsd supports it too).

Reply:
Sure - I thought about that, but feared it would break some obscure system where `copy_file_range` exists but does something else.
I've amended the patch to include a configure-time check. (And a fix for the CI `EOVERFLOW` failure, I hope.)